Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -696,6 +697,7 @@ public static Branch getAutoInstance(@NonNull Context context) {
         customReferrableSettings_ = CUSTOM_REFERRABLE_SETTINGS.USE_DEFAULT;
         boolean isTest = BranchUtil.checkTestMode(context);
         getBranchInstance(context, !isTest, null);
+        new BranchPreinstall().getPreinstallSystemData(context);
```

Comment:
Do we need to actually `new()` this class, or can the getPreinstallSystemData() be a static method?   By inspection it doesn't look like we are saving state anywhere.

Reply:
moved to static